test/hotspot/jtreg/runtime/cds/appcds/jigsaw/modulepath/OptimizeModuleHandlingTest.java

```diff
@@ -219,6 +219,16 @@ public static void runWithModulePath(String... extraRuntimeArgs) throws Exceptio
                        "-Xbootclasspath/a:", ".",
                        "--module-path", libsDir.toString(),
                        MAIN_CLASS)
+            .assertAbnormalExit(out -> {
+                out.shouldNotContain(CLASS_FOUND_MESSAGE)
+                   .shouldContain(OPTIMIZE_DISABLED)           // mapping info
+                   .shouldContain("Error: Could not find or load main class .");
```

Comment:
Is the fix changing the behavior for the test? Before fix, "." is seen as -Xbootclasspath/a:., but after fix, it is seen as 'main class'?

Reply:
I think the behavior after the fix is correct because there's a space between `-Xbootclasspath/a:` and '.'. So the '.'
shouldn't be treated as a path. I also added the following case 11 without the space in between.
`227         tty("11. run with CDS on, --module-path, with -Xbootclasspath/a:.");`